src/core/reducers/addons.js

```diff
@@ -17,6 +17,7 @@ export function denormalizeAddon(apiAddon) {
       ...apiAddon,
       // Set iconUrl to be consistent between disco and amo.
       iconUrl: apiAddon.icon_url,
+      is_restart_required: apiAddon.is_restart_required || false,
```

Comment:
This doesn't look like it's necessary since you already have one below in the `else if (thisAddon.current_version...` block.